tsdb/engine/tsm1/ring.go

```diff
@@ -103,28 +95,16 @@ func (r *ring) write(key []byte, values Values) (bool, error) {
 	return r.getPartition(key).write(key, values)
 }
 
-// add adds an entry to the ring.
-func (r *ring) add(key []byte, entry *entry) {
-	r.getPartition(key).add(key, entry)
-	atomic.AddInt64(&r.keysHint, 1)
```

Comment:
Note that since add is removed (it was unused in production code), keysHint is always 0